README.rst

```diff
@@ -38,10 +37,18 @@ Documentation
 
 http://python-visualization.github.io/folium/docs-master/
 
+Note that the documentation on Read the Docs is for the deprecated version
```

Comment:
Unfortunately I do not own the readthedocs account to remove that, so we need this note. Thanks!